src/MusicStore/Platform.cs

```diff
@@ -82,5 +82,14 @@ public bool IsRunningOnNanoServer
                 return _isNano.Value;
             }
         }
+
+        //Sql client not available on mono, non-windows, or nano
+        public bool UseInMemoryStore
```

Comment:
Reducing duplication is fine. All I'm saying is that this property should not be in this class